src/helpers.js

```diff
@@ -52,6 +56,62 @@ const inputCountriesToRuleCountries = inputCountries => {
     return ruleCountries
 }
 
+const getLevelOneCategory = cat =>
+    cat.split('-')[0]
+
+const getToFixedDecimal = num =>
+    parseFloat(num.toFixed(2))
+
+
+
+
+const getSuggestedPricingBoundsV1 = ({ minByCategory, countryTiersCoefficients, audienceInput }) => {
+    const { inputs } = audienceInput
+    const { location = {}, categories = {}, publishers = {}, advanced = {} } = inputs
+    const minCategoryCpm = Math.min(...Object.values(minByCategory))
+
+    const selectedCategoriesMinCpms = Object.values(Object.fromEntries(Object.entries(minByCategory).filter(([key, value]) => {
+
+        const inSelected = !categories.apply.includes('in') || !categories.in || !categories.in.length || categories.in.some(c => c === 'ALL' || (key === getLevelOneCategory(c)))
+        const notExcluded = !categories.apply.includes('nin') || !(categories.nin || []).some(c => key === getLevelOneCategory(c))
+
+        return inSelected && notExcluded
+    })))
+
+    const minCat = selectedCategoriesMinCpms.length ? Math.min(...selectedCategoriesMinCpms) : minCategoryCpm
+    const maxCat = selectedCategoriesMinCpms.length ? Math.max(...selectedCategoriesMinCpms) : minCategoryCpm
+
+    const selectedCountryCoefficients = Object.entries(CountryTiers).filter(([key, value]) => {
+        if (location.apply == 'allin') {
+            return true
+        }
+
+        const isInThisTier = (location[location.apply] || []).some(c => (c === key) || value.countries.includes(c))
+
+        if (location.apply == 'in') {
+            return isInThisTier
+        }
+        if (location.apply == 'nin') {
+            return !isInThisTier
+        }
+
+        return true
+    }).map(([key, _]) => countryTiersCoefficients[key])
+
+    const minTierCoefficient = Math.min(...Object.values(countryTiersCoefficients))
+    const minCountryCoef = selectedCountryCoefficients.length ? Math.min(...selectedCountryCoefficients) : minTierCoefficient
+    const maxCountryCoef = selectedCountryCoefficients.length ? Math.max(...selectedCountryCoefficients) : minTierCoefficient
```

Comment:
Shouldn't we have a `maxTierCoefficient` here at the end of the inline if which will come from a  `Math.max(...Object.values(countryTiersCoefficients))` ?